libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using prompts = Microsoft.Bot.Builder.Prompts;
+using static Microsoft.Bot.Builder.Prompts.PromptValidatorEx;
+using Microsoft.Bot.Schema;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Prompts;
+
+namespace Microsoft.Bot.Builder.Dialogs
+{
+    public class OAuthPromptSettingsWithTimeout : prompts.OAuthPromptSettings
+    {
+        public int? Timeout { get; set; }
+    }
+
+    public class OAuthPromptState : PromptOptions
```

Comment:
I don't understand how a base class can be "Options" and the derived class can be "state". 

These names seem like they should align.

Reply:
modified the name to be options isntead of state

---
In reply to: [184817336](https://github.com/Microsoft/botbuilder-dotnet/pull/520#discussion_r184817336) [](ancestors = 184817336)